core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html

```diff
@@ -66,11 +66,12 @@
          aria-haspopup="true"
          aria-expanded="false"
          aria-label="User Menu"
-         tabindex="0">
+         tabindex="0"
+         style="padding:10px 5px;">
```

Comment:
Why not add this to oppia-navbar-dropdown-toggle, if you want it in all pages? First make sure (by searching the codebase) that the class isn't used in other places, though.

Reply:
@seanlip  Padding is taken from nav >li >a { } from third_party.css .And the file is being  ignored by Git ,should I add it to Git.And adding to oppia-navbar-dropdown-toggle moves the Library and 
 About Link.